### ISSUE TITLE
Allow creation of multiple databases on the same QuestDB instance

### ISSUE THREAD

**Issue by rodryquintero** [questdb/questdb#4250]
### Is your feature request related to a problem?

Yes. If you spin a QuestDB server, in our case, we can only associate databases from a single instance/site of the database that exports data to questdb.

### Describe the solution you'd like.

 A _Namespace_ feature that allows creating different databases with the same table names would be useful. For example.

- ### QuestDB
  - **Database 1**
    - Orders
    - Clients
  - **Database 2**
    - Orders
    - Clients 

### Describe alternatives you've considered.

The only way to achieve this in the current version is to _prefix_ the tables with the database name. It works but it feels like a dirty workaround.

### Full Name:

ivan quintero

### Affiliation:

Healthcare

**Comment by rodryquintero** [comment]
> Hey @rodryquintero, thanks for raising this issue. Do you have any more information about your use case? Are you trying to support multi-tenancy?

Yes, multitenancy would be a nice thing to have. Not at the cloud level but on premise. We may have more than one application feeding data to the same QuestDB instance.

**Comment by newskooler** [comment]
If QuestDB were to have permissions on folder / sub-folder etc. structure, wouldn't that solve this issue?
